docs/Changelog.md

```diff
@@ -4633,6 +4633,43 @@ opset_import {
 <dd>Constrain input and output types to float tensors.</dd>
 </dl>
 
+### <a name="Shape-1"></a>**Shape-1**</a>
+
+  Takes a tensor as input and outputs an 1D int64 tensor with the shape of the input tensor.
```

Comment:
This might be unclear, maybe say `containing the shape of the input tensor` and give an example?

Reply:
I don't really like to put the example in the doc string because they render poorly. The short code snippets (from the test cases) serve much better as example.

Reply:
Wording updated